libcrux-ml-kem/tests/wycheproof.rs

```diff
@@ -0,0 +1,109 @@
+use libcrux_kats::wycheproof::mlkem::schema::*;
+
+use libcrux_ml_kem::{MlKemCiphertext, MlKemPublicKey};
+
+macro_rules! wycheproof_test {
+    ($name:ident, $parameter_set:expr, $module:path) => {
+        mod $name {
+            use super::*;
+
+            #[test]
+            fn wycheproof_keygen_and_decaps() {
```

Comment:
Since the test is called `wycheproof` already the prefixes seem unnecessary.
```suggestion
            fn keygen_and_decaps() {
```